python/pyspark/pandas/tests/indexes/test_base.py

```diff
@@ -2249,13 +2235,7 @@ def test_to_list(self):
         kmidx = ps.from_pandas(pmidx)
 
         self.assert_eq(kidx.tolist(), pidx.tolist())
-
-        if LooseVersion(pyspark.__version__) < LooseVersion("2.4"):
-            # PySpark < 2.4 does not support struct type with arrow enabled.
-            with self.sql_conf({SPARK_CONF_ARROW_ENABLED: False}):
-                self.assert_eq(kmidx.tolist(), pmidx.tolist())
-        else:
-            self.assert_eq(kidx.tolist(), pidx.tolist())
+        self.assert_eq(kidx.tolist(), pidx.tolist())
```

Comment:
Seems like the same tests?
I guess the second one should be `kmidx.tolist()`?

Reply:
Good catch! Updated.